File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    var answer = '';
    [...s].forEach((x,i)=>{
        if(s.charCodeAt(i)>=65 && s.charCodeAt(i)<=90){
            if(s.charCodeAt(i)+n>90) answer+=String.fromCharCode(s.charCodeAt(i)+n-26)
            else answer+=String.fromCharCode(s.charCodeAt(i)+n)
        }
        else if(s.charCodeAt(i)>=97 && s.charCodeAt(i)<=122)
            if(s.charCodeAt(i)+n>122) answer+=String.fromCharCode(s.charCodeAt(i)+n-26)
            else answer+=String.fromCharCode(s.charCodeAt(i)+n)
        else answer+=' '
    })
    return answer;
}

File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
#     아스키. ord, chr
    l1 = []
    for k in s:
        if k == ' ':
            l1.append(' ')
            continue
        #대문자란뜻
        if ord(k)<=90:
            shift = ord(k)+n
            if shift >90:
                shift-=26
        else:
            shift = ord(k)+n
            if shift > 122:
                shift-=26

        l1.append(chr(shift))
    answer = ''.join(l1)
    return answer